locales/hu.js

```
@@ -1,137 +1,245 @@
+    "Június",
+    "Július",
+    "Augusztus",
+    "Szeptember",
+    "Október",
+    "November",
+    "December",
+  ],
+
+  // Bookmarks
+  bookmarksHeading: "Könyvjelzők",
+  bookmarkViewAs: "Megjelenítés mint",
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)